dGame/dMission/Mission.cpp

```diff
@@ -47,6 +48,7 @@ Mission::Mission(MissionComponent* missionComponent, const uint32_t missionId) {
 
         m_Tasks.push_back(task);
     }
+    m_CurrencyConversionRate = dZoneManager::Instance()->GetLevelCapCurrencyConversion();
```

Comment:
Do not store this value in the Mission class, retrieve it every time it's used. Storing it here is redundant.

Reply:
Removed